codemod/src/plugins/replace-typography-tokens/index.ts

```diff
@@ -0,0 +1,166 @@
+import * as postcss from "postcss";
```

Comment:
This plugin will replace font tokens with the typography mixins with the matched params, and if there is usage with combined font-size, font-weight and line-height with the matched params it will replace it with the match mixin